spec/features/projects/viewing_project_spec.rb

```diff
@@ -54,8 +54,9 @@
         click_link "Temporal"
         fill_in "Start", with: "2010-01-01 00:00:00\t\t"
         fill_in "End", with: "2014-02-01 00:00:00\t\t"
-        js_click_apply ".temporal-dropdown"
         wait_for_xhr
+        click_button('Apply Filter')
+        sleep 1
```

Comment:
I know 'sleep' isn't a good option - however, I'm just trying to find *something* that works here.  'wait_for_xhr' is throwing an odd error.

Reply:
Yea, this isn't something we should consider a solution though. This test will fail just as 'reliably' locally when run. This particular test failure is an actual bug, there are things going on in the background of this page (more specifically viewing the project page for a collection that the user doesn't have access to view) which prevents the page from loading. At least this was my initial thought -- so, 

1. Let's try to get tests to pass reliably locally instead of pushing changes to GitHub in hopes that they pass, if we can't trust the tests then they are useless.

2. We should always run tests locally before pushing, a PR isn't a holding area for our code, we should use PRs as a place to review completed code.

Reply:
This particular failure is also plaguing the other open PR, so I'd save investigation of that for last on the ticket.. something screwy going on :-/

Reply:
@abbottry - I found a way around this failure by doing an 'expect' on the address bar URL to ensure that the temporal range setting had 'taken'.  No more 'sleep'!   

Not sure if this solution helps the problem with your PR or not, but you might want to review to consider it.